### ISSUE TITLE
Update runtime to GNOME 46

### ISSUE THREAD

**Issue by x80486** [flathub/org.inkscape.Inkscape#165]

**Comment by Erick555** [comment]
@hfiguiere looks like this update wasn't published in flathub:

```
$ flatpak remote-info flathub org.inkscape.Inkscape

Inkscape - Vector Graphics Editor

        ID: org.inkscape.Inkscape
       Ref: app/org.inkscape.Inkscape/x86_64/stable
      Arch: x86_64
    Branch: stable
   Version: 1.3.2
   License: GPL-2.0-or-later
Collection: org.flathub.Stable
  Download: 103.0 MB
 Installed: 305.8 MB
   Runtime: org.gnome.Platform/x86_64/45
       Sdk: org.gnome.Sdk/x86_64/45

    Commit: 83a8115af2ae077cb3e126422a1a5cf444a5cd7664a3ac7a5be754cd2b240094
    Parent: 51bcc03490f02d12986e168f44782199b822dcade5a1e0e01d547dae47c0b314
   Subject: Fix appdata to pass validation (3e5cb8a3)
      Date: 2024-03-02 20:52:45 +0000

```

**Comment by hfiguiere** [comment]
looks like https://github.com/flathub/flathub/issues/5225 hit again.
